coderd/autobuild/lifecycle_executor.go

```diff
@@ -325,14 +320,24 @@ func (e *Executor) runOnce(t time.Time) Stats {
 						return xerrors.Errorf("post provisioner job to pubsub: %w", err)
 					}
 				}
-				if dormantNotification != nil {
-					_, err = dormancy.NotifyWorkspaceDormant(
+				if shouldNotifyDormancy {
+					_, err = e.notificationsEnqueuer.Enqueue(
 						e.ctx,
-						e.notificationsEnqueuer,
-						*dormantNotification,
+						ws.OwnerID,
+						notifications.TemplateWorkspaceDormant,
+						map[string]string{
+							"name":           ws.Name,
+							"reason":         "prolonged inactivity, exceeding the dormancy threshold",
```

Comment:
Shortened this. WDYT?

```suggestion
							"reason":         "inactivity exceeded the dormancy threshold",
```